app/frontend/src/features/messages/groupchats/GroupChatView.tsx

```diff
@@ -36,11 +73,20 @@ export default function GroupChatView() {
         <CloseIcon />
         (leave)
       </Button>
-      {error && <Alert severity="error">{error}</Alert>}
-      {loading ? (
+      {(error || sendMutation.error || leaveGroupChatMutation.error) && (
+        <Alert severity="error">
+          {error ||
+            sendMutation.error?.message ||
+            leaveGroupChatMutation.error?.message}
+        </Alert>
+      )}
+      {isLoading ? (
         <CircularProgress />
       ) : (
-        <MessageList messages={messages} handleSend={handleSend} />
+        <>
+          <MessageList messages={messages!} />
+          <SendField mutation={sendMutation} />
```

Comment:
```suggestion
          <SendField sendMutation={sendMutation} />
```
Probably ok for this, but I find keeping props consistent a bit easier for tracing the code, especially when props are being passed around multiple layers.